packaging/windows/WINDOWS_INSTALLER.md

```diff
@@ -6,55 +6,35 @@ Netdata offers a convenient Windows installer for easy setup. This executable pr
 >
 > This feature is currently under beta and only available for Nightly releases, and the installer can be found in our [nightlies repo](https://github.com/netdata/netdata-nightlies). A stable version will be released soon.
 
-## Graphical User Interface (GUI)
-
-Double-clicking the installer initiates the setup process. Since Netdata adds a service to your system, you'll need to provide administrator privileges.
-
-The installer will then guide you through these steps:
-
-1. **Welcome**: This screen provides a summary of the actions the installer will perform.
-2. **License Agreements**:
-    - [Netdata Cloud UI License](/src/web/gui/v2/LICENSE.md): Review and accept the license terms to proceed.
-    - [GPLv3 License](/LICENSE): Read the GNU General Public License v3, which governs the Netdata software.
-3. **Destination**:  Choose the installation directory. By default, Netdata installs in `C:\Program Files\Netdata`.
-4. **Installation**: The installer will copy the necessary files to the chosen directory.
-5. **Connect to Netdata Cloud**: To [connect](/src/claim/README.md) your Agent to your Netdata Cloud Space you need to provide the following:
-    - **Claim Token**: The Claim Token that securely authenticates and links your Agent to your Space.
-    - **Room IDs**: A comma-separated list of Room IDs where you want to add your Agent.
-    - **Proxy address**: The address of a proxy server, if one is required for communication with Netdata Cloud.
-    - **Insecure connection**: By default, Netdata verifies the server's certificate. Enabling this option bypasses verification (use only if necessary).
-    - **Open Terminal**: Select this option to launch the `MSYS2` terminal after installation completes.
-6. **Finish**: The installation process is complete!
-
 ## Silent Mode (Command line)
 
-This section provides instructions for installing Netdata in silent mode, which is ideal for automated deployments.
+This section provides instructions for installing Netdata in silent mode running `msiexec.exe`, which is ideal for automated deployments.
 
 > **Info**
 >
 > Run the installer as admin to avoid the Windows prompt.
 >
-> Silent mode skips displaying license agreements, but requires explicitly accepting them using the `/A` option.
+> Silent mode skips displaying license agreements, but requires explicitly accepting them using the arguments `GPLLICENSE=1` and `CLOUDLICENSE=1`.
```

Comment:
There is no Cloud license. I asked to rename it multiple times. @stelfrag can you help us please?

Reply:
Renamed.